workspaces/ui/public/index.html

```diff
@@ -46,7 +46,7 @@
   To create a production bundle, use `npm run build` or `yarn build`.
 -->
 <script>
-    window.opticAnalyticsEnabled = '%REACT_APP_ENABLE_ANALYTICS%' !== 'no';
+    window.opticAnalyticsEnabled = '%REACT_APP_ENABLE_ANALYTICS%' === 'yes';
```

Comment:
x !== 'no' means enabled by default, opt-out with 'no'. x === 'yes' means disabled by default, opt-in.